config/config.go

```diff
@@ -50,6 +50,8 @@ var format = logging.MustStringFormatter(
 
 // Config defines the central type where all configuration is umarhsalled to.
 type Config struct {
+	TomlMetadata toml.MetaData
```

Comment:
What about having Config embed toml.MetaData, like this:

```
type Config struct {
toml.MetaData
}
```

Reply:
Makes sense, I fixed it.